curriculum/challenges/english/25-front-end-development/quiz-css-typography/66ed9010f45ce3ece4053eb8.md

```diff
@@ -39,73 +39,73 @@ Which of the following is the correct way to write a `@font-face` rule in CSS?
 
 #### --text--
 
-What is the purpose of the offset values in the `text-shadow` property, and how do they affect the shadow's appearance?
+How do the offset values in the CSS `text-shadow` property affect the shadow's appearance?
 
 #### --distractors--
 
-They adjust the text's size and weight
+They adjust the weight and color of the shadow.
 
 ---
 
-They change the shadow color
+They change the color of the shadow.
 
 ---
 
-They remove text from the shadow's path
+They remove the text from the path of the shadow.
 
 #### --answer--
 
-They control the horizontal and vertical position of the shadow relative to the text
+They control the position of the shadow relative to the text.
 
 ### --question--
 
 #### --text--
 
-What does "kerning" mean in typography?
+What does kerning refer to in the context of typography?
 
 #### --distractors--
 
-The space between lines of text
+How the space between lines of text is adjusted to improve readability.
```

Comment:
```suggestion
How the space between lines of text is adjusted to improve readability and aesthetics.
```

Just trying to lengthen this a bit so the correct answer isn't so much longer than the others.